subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/CheckstylePluginVersionIntegrationTest.groovy

```diff
@@ -202,6 +201,7 @@ class CheckstylePluginVersionIntegrationTest extends MultiVersionIntegrationSpec
         then:
         file("build/reports/checkstyle/main.html").exists()
         !file("build/reports/checkstyle/main.xml").exists()
+        file("build/tmp/checkstyleMain/main.xml").exists()
```

Comment:
I'd prefer that the plugin deletes the XML report in the temporary directory after executing the checkstyle task. What the user says here is: Generate the HTML report but not the XML report. I'd hate to see the `build` directory being filled up by temp. files just because they are required to produce a different output.